app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
```

Comment:
Please be patient with sort of imports. Built-in firstly, then blank line, you own packages at the last place.